### ISSUE TITLE
"cbNameLater = Earth" still an issue

### ISSUE THREAD

**Issue by Gordon-Dry** [KSP-RO/RealSolarSystem#170]
I quote
https://forum.kerbalspaceprogram.com/index.php?/topic/50533-140-16x-module-manager-402-february-3rd-2019-right-to-ludicrous-speed/&page=251&tab=comments#comment-3539665

> File https://github.com/KSP-RO/RealSolarSystem/blob/v14.0.0/GameData/RealSolarSystem/RSSKopernicus/Earth/Earth.cfg has the line "cbNameLater = Earth" that needs to be killed. "Template {name=Kerbin}" is ok and so is "Properties {displayName = Earth}". cbRenameLater has been deprecated and now all the more recent mods use Kerbin as the primary's name in their internal config. @Galileo did it for GPP a few months back. I had to go through a history of "Couldn't find CelestialBodyName Kerbin" log errors and also Gael errors in order to understand the stuttering problem. Had to go through the forum history for every mention of that error. It took me a week to figure it out.

**Comment by lamont-granquist** [comment]
cross linking https://github.com/jrossignol/WaypointManager/issues/59

**Comment by Gordon-Dry** [comment]
Yes, removing Waypoint Manager alone helps in avoiding the log spamming.

**Comment by PhineasFreak** [comment]
I'd love to see some evidence about this parameter being deprecated/old/bad/killing puppies.

**Comment by Gordon-Dry** [comment]
Not sure. If "just" Waypoint Manager is bad.

And if some memory of last year or even earlier is obsolete now - that cbNameLater alone created issues.

I suggest to let this stay open because of the cross linking until Waypoint Manager is fixed.

**Comment by PhineasFreak** [comment]
This issue is fixed and will be part of a RSS release in the near future (#320). Closing.
